app/models/analytics-value-service-query-fallbacks.js

```diff
@@ -203,8 +203,10 @@ class Fallbacks {
         this.flowAggregate = {
             metrics: [
                 "nFlow",
+                "nFlowMilestone",
```

Comment:
@WitLi07 aren't these values supposed to be pulled dynamically from the swagger definition? This list should only be a fallback IIRC. If that's true, a report of missing metrics means we're failing to load from swagger and using the fallback instead, and need to fix that.

Reply:
If there's no existing process for _these specific_ data points to be loaded from swagger, this is fine as is; we don't need to implement new functionality at this time.

Reply:
@timsmithgenesys I added to fallbacks here, but the main file I changed was the `publicapi-v2-latest.json` file.  Changes to this fallback file weren't reflecting on my local instance, though.

Reply:
@timsmithgenesys Yes that should be the case. If it's dynamically loading from the swagger definition, then changing the fallback file should not affect what's actually displayed the UI, IIRC. 
@diemenschmaschine92 Since you added those metrics in the swagger definition, can you verify that if you remove those metrics in the fallback file, are they still present?

Reply:
Reviewed with Jacob. We found that the swagger service, and by extension the analytics query builder, was still attempting to load the swagger definition from a route that used to exist on the old dev center's domain. That route was a passthrough to serve the swagger definition because the API didn't apply CORS headers to it originally. Today, that route has been removed from the new dev center, but it's no longer necessary since the API does apply CORS headers to the swagger response. The remediation for this issue is to update the swagger service to use updated logic to determine the correct URL from which to load the swagger definition. This logic already exists for determining the URL to pass to the embedded open api explorer: https://github.com/MyPureCloud/developer-tools/blob/a202edb05d464421df182b466a1f34c836828d3c/app/routes/api-explorer.js#L43